src/Slices/ResourceDiscovery/UI/Components/ManagedResourceLink.tsx

```diff
@@ -4,19 +4,20 @@ import { ResourceLink } from "@/UI/Components";
 
 interface Props {
   resourceUri: string | null;
+  type: "managed" | "discovery";
 }
 
 /**
- * @Component that renders a link to the managed resource
+ * @Component that renders a link to the managed or the discovery resource
  * or a "-" if the resourceUri is null or empty.
  *
- * managed_resource_uri comes in format : /api/v2/resource/<rid>
+ * uris comes in format : /api/v2/resource/<rid>
  *
- * @param resourceUri : API URI of the managed resource
+ * @param resourceUri : API URI of the managed/discovery resource
  *
  * @returns ManagedResourceLink component
  */
-export const ManagedResourceLink: React.FC<Props> = ({ resourceUri }) => {
+export const ManagedResourceLink: React.FC<Props> = ({ resourceUri, type }) => {
```

Comment:
Right now, the name of the component suggests it only is for one type. So, maybe we should rename it to something more meaningful. `ResourceLink` is perhaps too generic, and we might confuse it with other similar components. But maybe that's fine 🤔